### DISCUSSION TITLE
传入表单多余的数据能在获取表单的数据的时候（比如submit的回调或者form.values）去掉吗

### DISCUSSION THREAD

**Discussion by zhousufa** [alibaba/formily#3162] [Q&A 2.x]
createForm的时候传入values {"a":1,"b":2,"c":3},但是schema只有a和b，就是说c是不存在的，目的是在获取表单的数据的时候（比如submit的回调或者form.values）获取的数据是a和b，这个能做到吗

**Comment by shaoqiwang-fe** [comment]
mark

**Comment by li9269391** [comment]
https://github.com/alibaba/formily/discussions/1637  之前有讨论过。但仍然觉得有这个必要，如果为了 form 模型能挂载其它数据，应该像 field 模型增加一个扩展的 data 属性。但 form.values 应该是与 field 关联，做过滤再输出
